internal/db/schema/migrations/postgres/16/04_plugin_host.up.sql

```diff
@@ -129,6 +129,7 @@ begin;
     attributes bytea not null
       constraint attributes_must_not_be_empty
         check(length(attributes) > 0),
+    preferred_endpoints text,
```

Comment:
Is this a list?

Reply:
Yes, it is a delimited list.

Reply:
Resolving for now while I work on the other approach.